atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -75,7 +75,9 @@ protected TransactionManager getManager() {
                 Suppliers.ofInstance(AtlasDbConstraintCheckingMode.FULL_CONSTRAINT_CHECKING_THROWS_EXCEPTIONS),
                 conflictDetectionManager,
                 SweepStrategyManagers.createDefault(keyValueService),
-                NoOpCleaner.INSTANCE);
+                NoOpCleaner.INSTANCE,
+                GET_RANGES_CONCURRENCY,
```

Comment:
Where do these constants live? (it looks a little weird without e.g. `AbstractTransactionTest.CONSTANT`)

(edit: answered myself by scrolling down a bit. but it still looks weird)